evap/staff/templates/staff_course_form.html

```diff
@@ -85,7 +85,7 @@
 
```

Comment:
what about the "You are editing a course which has not been approved by the editor yet." warning? i'd expect that to show up on the course create page.